test/test_helper.exs

```diff
@@ -25,4 +27,32 @@ defmodule Protobuf.TestHelpers do
 
     Map.put(context, :tmp_dir, tmp_dir_name)
   end
+
+  def get_type_spec_as_string(module, bytecode, type)
+      when is_atom(module) and is_binary(bytecode) and is_atom(type) do
+    assert {:ok, {_, [debug_info: {:debug_info_v1, backend, data}]}} =
+             :beam_lib.chunks(bytecode, [:debug_info])
+
+    assert {:ok, specs} =
+             (case data do
+                {:elixir_v1, %{}, specs} ->
+                  {:ok, specs}
+
+                _ ->
+                  case backend.debug_info(:erlang_v1, module, data, []) do
+                    {:ok, abstract_code} -> {:ok, abstract_code}
+                    _ -> :error
+                  end
```

Comment:
I don't think we need this since we are not generating erlang modues :)

Reply:
Aaah, nice! Good catch. I just copied this code over from Elixir 🙈